Week-1/index.html

```diff
@@ -0,0 +1,117 @@
+          <div class="header-section">
+            <img
+              src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"
+            />
+            <div class="header-inside">
+              <div>
+                <h1>CattyBoard Top 100 Single Charts(11.06.36)</h1>
+                <p>Unknown Artist</p>
+                <p>2016.Charts.100 songs</p>
+              </div>
+              <div class="header-down">
+                <div>
+                  <p>
+                    <i class="fa fa-play" aria-hidden="true"></i>&nbsp Play all
+                  </p>
+                </div>
+                <div>
+                  <p>
+                    <i class="fa fa-plus" aria-hidden="true"></i> &nbspAdd to
+                  </p>
+                </div>
+                <div>
+                  <p>
+                    <i class="fa fa-ellipsis-h" aria-hidden="true"></i>
+                    &nbspMore
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </header>
+        <div class="main-page">
+          <div class="main-div-1">
+            <p class="main-text">1.One Dance</p>
+            <p class="main-text">Crake feat CatKid & Cyla</p>
+            <p class="main-text">2:54</p>
+            <button class="main-spec">CATTY CLOUD SYNC</button>
+          </div>
+          <div class="div-2">
+            <div class="main-div-2">
+              <p class="main-text">2.Panda</p>
+              <p class="main-text">Catte</p>
+              <p class="main-text">4:06</p>
+              <button class="main-spec">CATTY CLOUD SYNC</button>
+            </div>
+          </div>
+
+          <div class="main-div-1">
+            <p class="main-text">3.Can't Stop the Feeling</p>
+            <p class="main-text">Catlin Cimberlake</p>
+            <p class="main-text">3:56</p>
+            <button class="main-spec">CATTY CLOUD SYNC</button>
+          </div>
+          <div class="div-2">
+            <div class="main-div-2">
+              <p class="main-text">4.Work From Home</p>
+              <p class="main-text">Cat Harmony featColla</p>
+              <p class="main-text">3:34</p>
+              <button class="main-spec">CATTY CLOUD SYNC</button>
+            </div>
+          </div>
+        </div>
+      </section>
+    </div>
+    <footer class="page-footer">
+      <div class="footer-inside">
+        <img
+          src="https://vif1g.csb.app/src/assets/cattyboard.jpg"
+          class="footer-img"
+        />
+        <div class="footer-name">
+          <p class="footer-text">If It Ain't Love</p>
+          <p>Caston Derulo</p>
+        </div>
+      </div>
+      <div class="footer-icon">
+        <i class="fa fa-step-backward" aria-hidden="true"></i>
+        <i class="fa fa-pause" aria-hidden="true"></i>
+        <i class="fa fa-step-forward" aria-hidden="true"></i>
+        <i class="fa fa-undo" aria-hidden=""></i>

```

Comment:
if no value is given to aria-hidden="", donot use it

Reply:
ok mam